ruby-gem/Rakefile

```diff
@@ -65,19 +22,72 @@ or set CALABASH_SERVER_PATH to point to your local copy of the server.
 
 $ CALABASH_SERVER_PATH=/path/to/server bundle exec rake build
 
-For full instuctions see: https://github.com/calabash/calabash-android/wiki/Building-calabash-android\033[0m
+For full instuctions see: https://github.com/calabash/calabash-android/wiki/Building-calabash-android
+
+]
+        end
+        calabash_server_dir
+      end
+
+      def self.test_server_version
+        test_server_version_path = File.join(self.find_server_repo_or_raise, "version")
+        File.read(test_server_version_path).chomp
+      end
+
+      def expect_versions_aligned
+        if ENV["SKIP_VERSION_CHECK"] == "1"
+          puts "Skipping gem/server version check"
+        else
+          test_server_dir = File.join(self.find_server_repo_or_raise, "server")
 
-    }
+          if test_server_version != Calabash::Android::VERSION
+            raise(%Q[
+Expected server version:
+
+    #{test_server_version}
+
+to match gem version:
+
+    #{Calabash::Android::VERSION}
+
+])
+          end
+        end
+      end
+
+      def self.build_test_apk
+        test_server_dir = File.join(self.find_server_repo_or_raise, "server")
+        Dir.chdir(test_server_dir) do
+          system('BUILDOUTPUT="$(./gradlew clean assembleAndroidTest -q)"')
```

Comment:
I added this before to remove noise from build command logs. Now we have separate command to build apk from here, so it should be ok to see full build output:
` system('./gradlew clean assembleAndroidTest')`